dist/yaml/ovn-setup.yaml

```diff
@@ -0,0 +1,189 @@
+# ovn-setup.yaml
+#
+# Setup for Openshift to support the ovn-cni plugin
+#
+# Create the project, service account and policies.
+# ovnkube interacts with kubernetes and the environment
+# must be properly set up.
+# 
+# This provisioning is done as part of installation after the cluster is
+# up and before the ovn daemonsets are created.
+
+---
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+    openshift.io/node-selector: ""
+    openshift.io/sa.scc.mcs: s0:c8,c2
+    openshift.io/sa.scc.supplemental-groups: 1000060000/10000
+  name: ovn-kubernetes
+spec:
+  finalizers:
+  - openshift.io/origin
+  - kubernetes
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ovn
+  namespace: ovn-kubernetes
+
+---
+apiVersion: authorization.openshift.io/v1
```

Comment:
All of the authorization.openshift.io/v1 should instead be "rbac.authorization.k8s.io/v1"